spyder/plugins/tests/test_history.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+
+import pytest
+
+from qtpy.QtGui import QTextOption
+
+from spyder.plugins import history
+
+
+#==============================================================================
+# Constants
+#==============================================================================
```

Comment:
Please remove this block comment if it's not needed.

Reply:
Done.